.github/workflows/release.yml

```diff
@@ -102,14 +102,14 @@ jobs:
           title: 'chore: 🤖 update versions'
           publish: pnpm release:publish
           version: pnpm release:version
-          createGithubReleases: false
+          createGithubReleases: true
```

Comment:
defaults to true 
https://github.com/changesets/action?tab=readme-ov-file#inputs